snapcraft.yaml

```diff
@@ -37,16 +41,16 @@ parts:
     source: https://gitlab.gnome.org/GNOME/gnome-mahjongg.git
     source-type: git
     source-tag: '3.38.3'
+    parse-info: [usr/share/metainfo/org.gnome.Mahjongg.appdata.xml]
     override-pull: |
-      snapcraftctl pull
-      snapcraftctl set-version $(git describe --tags --abbrev=10)
+      craftctl default
+      craftctl set version=$(git describe --tags --abbrev=10)
+      patch -p1 < $CRAFT_PROJECT_DIR/patches/patch1.diff
```

Comment:
Please add a comment at the top of the patch saying what it is for. Also please give it a more intuitive name than patch1.diff. Also use *.diff here.

Reply:
Done.